docs/griptape-framework/drivers/prompt-drivers.md

```diff
@@ -56,15 +56,10 @@ print(result.value)
 
 Griptape offers the following Prompt Drivers for interacting with LLMs.
 
-!!! warning
-    When overriding a default Prompt Driver, take care to ensure you've updated the Structure's configured Embedding Driver as well. If Task Memory isn't needed, you can avoid compatibility issues by setting `task_memory=None` to disable Task Memory in your Structure.
-
 ### OpenAI Chat
 
-!!! info
-    This driver uses [OpenAi function calling](https://platform.openai.com/docs/guides/function-calling) when using [Tools](../tools/index.md). You can change this to use Griptape's own tool calling implementation by setting `use_native_tools` to `False`.
-
 The [OpenAiChatPromptDriver](../../reference/griptape/drivers/prompt/openai_chat_prompt_driver.md) connects to the [OpenAI Chat](https://platform.openai.com/docs/guides/chat) API.
+This driver uses [OpenAi function calling](https://platform.openai.com/docs/guides/function-calling) when using [Tools](../tools/index.md).
```

Comment:
Does tools/index.md call out `use_native_tools=False` as an option?

Reply:
[It does now](https://github.com/griptape-ai/griptape/pull/994/files#diff-968a4b3b3f733a9b6996cfd71b083d7b50a68dd8db9bf8ea3ba6f9cafc091aff), take another gander